src/components/SignInButton.tsx

```diff
@@ -1,11 +1,23 @@
 // importing from next-auth/react instead of our root's auth.ts because of Client & Server component issues.
 // https://authjs.dev/getting-started/session-management/login
 import { signIn } from 'next-auth/react';
+import { Button } from '@chakra-ui/react';
+import { FcGoogle } from 'react-icons/fc';
 
 export function SignInButton() {
   return (
-    <button onClick={() => signIn('google', { callbackUrl: `/homepage` })}>
-      Sign In Google
-    </button>
+    <Button
+      variant="outline"
+      borderColor="blue.400"
+      colorScheme="gray"
+      width="275px"
+      height="45px"
+      color="black"
```

Comment:
```suggestion
      color="text"
```
**Before:**
<img width="383" alt="image" src="https://github.com/user-attachments/assets/1bf05d5f-9aad-49d0-9db8-9754694e0fe1">

**After:**
<img width="355" alt="image" src="https://github.com/user-attachments/assets/cba59e03-4bee-4297-bf15-a2dbfa125380">